Makefile

```diff
@@ -86,9 +86,12 @@ manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
 
 # Run go fmt against code
-fmt:
+fmt: groupimports
 	$(GO) fmt ./...
 
+groupimports:
```

Comment:
Did we need to install goimport first

Reply:
Yes! Though `goimports` is auto installed as `golang.org/x/tools` is an indirect dependency, it's still necessary to check and install.